src/app/meetups/meetups.component.ts

```diff
@@ -66,6 +93,38 @@ export class MeetupsComponent implements OnInit, AfterViewInit {
         }, (error) => this.deleteDialog.componentInstance.message = 'There was a problem deleting this meetup');
     };
   }
+  deleteSelected() {
+    let amount = 'many',
+      okClick = this.deleteMeetup(this.selection.selected),

```

Comment:
`this.selection.selected` is an array, but `this.deleteMeetup` is for a single object with `_rev` and `_id` properties.